www/data/docs_sidebar.yml

```diff
@@ -204,5 +204,14 @@ sidebar_links:
       link: "/docs/internals/internals/#crypto-internals"
     - title: Bootstrapping Chef Habitat
       link: "/docs/internals/internals/#bootstrap-internals"
+  - title: Habitat Pattern Library
+    link: "/docs/pattern-library/pattern-library"
+    sub_links:
+    - title: Plan.sh Pattern
```

Comment:
I think it might be worth grouping these under a Kubernetes heading to indicate what the purpose is.  "plan.sh pattern" on its own is pretty generic and could apply to a number of things.  Perhaps: `Kubernetes bastion service plan.sh`?